caas/kubernetes/provider/pebble/pebble.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package pebble
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Probe constants
+const (
+	alivePath = "/v1/health?level=alive"
+	readyPath = "/v1/health?level=ready"
+)
+
+func LivenessHandler(port string) corev1.ProbeHandler {
+	return corev1.ProbeHandler{
+		HTTPGet: &corev1.HTTPGetAction{
+			Path: alivePath,
+			Port: intstr.Parse(port),
+		},
+	}
+}
+
+func ReadinessHandler(port string) corev1.ProbeHandler {
+	return corev1.ProbeHandler{
+		HTTPGet: &corev1.HTTPGetAction{
+			Path: readyPath,
+			Port: intstr.Parse(port),
+		},
+	}
+}
+
+// Reserved Pebble health check ports for each container.
+// All ports from 38813 onwards are reserved for workload containers
+// Additional ports should be allocated backwards
+// e.g. 38812, 38811, 38810, ...
+const (
+	// for api-server container (jujud)
```

Comment:
Can we merge these const with the ones above?

Reply:
this block is just for port numbers, want to keep them separate